assets/_scss/core/_defaults.scss

```diff
@@ -0,0 +1,52 @@
+// Starter Kit Defaults
+// Do not modify.
+// These variables can be modified in _variables.scss
+
+// Typography
+$em-base:       10px; // default is already defined in bourbon
+$base-font-size:    rem(16px) !default;
+$small-font-size:   rem(14px) !default;
+$title-font-size:   rem(48px) !default;
+$h1-font-size:      rem(36px) !default;
+$h2-font-size:      rem(28px) !default;
+$h3-font-size:      rem(24px) !default;
+$h4-font-size:      rem(18px) !default;
+$h5-font-size:      rem(16px) !default;
+$h6-font-size:      rem(14px) !default;
+$base-line-height:    1.5em !default;
+$heading-line-height: 1.3em !default;
+
+$font-sans:       'Source Sans Pro', 'Helvetica', 'Arial', sans-serif !default;
+$font-serif:      'Georgia', 'Times New Roman', serif;
+$font-mono:       'Source Code Pro', 'Courier New', 'Courier', monospace !default;
```

Comment:
Unless you are planning on using that variable elsewhere, I'd say removing that variable sounds like a good solution.